checker/src/main/java/org/checkerframework/checker/determinism/DeterminismAnnotatedTypeFactory.java

```diff
@@ -275,9 +275,9 @@ public Void visitMemberSelect(
 
         /**
          * Reports an error if {@code node} represents explicitly constructing a {@code @Det} or
-         * {@code @PolyDet} {@code HashSet}. If one these annotation wasn't explicitly written, but
```

Comment:
Should it say `{@code @Det HashSet}` instead of `{@code @Det}` or `{@code HashSet}`?